Firebase/android/core/src/main/java/com/novoda/bonfire/channel/presenter/ChannelsPresenter.java

```diff
@@ -48,6 +59,7 @@ public void call(Authentication authentication) {
                     }
                 })
                 .flatMap(channelsForUser())
+                .map(sortIfConfigured())
```

Comment:
The Firebase config is handling it automatically, there is a cache time parameter on the init and it will update if the cache is expired.